packages/components/Tabs/src/TabsItem.tsx

```diff
@@ -64,7 +67,7 @@ export const TabsItem = compose<TabsItemType>({
       info: {
         ...pressable.state,
         selected: info.selectedKey === userProps.itemKey,
-        icon: !!icon,
+        icon: !!icon && Platform.OS !== 'windows', // Icons are on backlog for windows, GH 935
```

Comment:
Update the other GH issue comments as well.  Also put the GH issue first, I.E
> GH #935, Icons are on backlog for windows
Not sure I have a good reason for putting the issue first except that it's the first thing the reader sees then, and is convention I've seen elsewhere.